TASVideos/Pages/Forum/Topics/_TopicPost.cshtml

```diff
@@ -68,7 +68,7 @@
 									<small condition="@(Model.PosterPronouns != PreferredPronounTypes.Unspecified)" class="text-body-tertiary col-auto col-md-12" style="line-height: 22px;">@Html.DisplayFor(m => m.PosterPronouns)</small>
 								</row>
 								<div class="ms-2 ms-md-0" style="line-height: 1">
-									<small>@string.Join(", ", Model.PosterRoles.OrderBy(s => s).Append(Model.PosterPlayerRank).Where(s => !string.IsNullOrEmpty(s)).Select(s => s!.Replace(' ', '\u00A0')))
+									<small>@string.Join(", ", Model.CalculatedRoles)
```

Comment:
This seems to change things quite a bit. 
1. The roles now don't have a space between them anymore `Editor,Judge,Expert player` instead of `Editor, Judge, Expert player`.
2. The second string.Join is not necessary anymore.
3. Banned users will still show their points, but not their rank, `Banned User (534)` which seems weird.

Reply:
fixed those 3 things